core/util/webhunter.go

```diff
@@ -608,7 +608,6 @@ func execute(req *http.Request, baseReq *Request) (*Result, error) {
 		reader, err = gzip.NewReader(resp.Body)
 
 		if err != nil {
-			apm.CaptureError(ctx, err).Send()
```

Comment:
Oh neat, do these get handled now because of using ctx = apm.ContextWithTransaction(...)?

Reply:
I moved it to the joint and pipeline level. That way we can control what errors propagate up to apm. So the fetch joint should now send that error if all the retries fail.